userguide/content/en/docs/adding-content/diagrams-and-formulae/index.md

```diff
@@ -422,9 +437,9 @@ params:
 {{< /tab >}}
 {{< /tabpane >}}
 
-See the [Mermaid documentation](https://mermaid-js.github.io/mermaid/#/Setup?id=mermaidapi-configuration-defaults) for a list of defaults that can be overridden.
+See the [Mermaid source code](https://github.com/mermaid-js/mermaid/blob/master/packages/mermaid/src/defaultConfig.ts#L18) for a list of defaults that can be overridden.
```

Comment:
I'd rather we use the link to the documentation, as it was before.

Reply:
Reverted that passage, as requested.